examples/profile-client/src/main/java/com/palantir/example/profile/schema/generated/UserPhotosStreamStore.java

```diff
@@ -100,7 +100,7 @@ protected void storeBlock(Transaction t, long id, long blockNumber, final byte[]
             touchMetadataWhileStoringForConflicts(t, row.getId(), row.getBlockId());
             tables.getUserPhotosStreamValueTable(t).putValue(row, block);
         } catch (RuntimeException e) {
-            log.error("Error storing block " + row.getBlockId() + " for stream id " + row.getId(), e);
+            log.error("Error storing block {} for stream id {}", row.getBlockId(), row.getId(), e);
```

Comment:
Unfortunately, this has been changed in the generated code. Will revert and fix.